types.d.ts

```diff
@@ -0,0 +1,109 @@
+import type { ArchiverOptions } from 'archiver';
```

Comment:
I don't think archiver has published types. we may need to add a dependency `@types/archiver`

Reply:
Added to devDependencies.

Reply:
It should be a dependency. the package manager won't install devDependencies during `npm/yarn install <package>`